command/agent/config.go

```diff
@@ -711,7 +816,10 @@ func DecodeConfig(r io.Reader) (*Config, error) {
 	// use mapstructure decoding, so we need to account for those as well.
 	allowedKeys := []string{
 		"service", "services", "check", "checks", "statsd_addr", "statsite_addr", "statsite_prefix",
-		"dogstatsd_addr", "dogstatsd_tags",
+		"dogstatsd_addr", "dogstatsd_tags", "circonus_api_token", "circonus_api_app",
+		"circonus_api_url", "circonus_submission_url", "circonus_submit_interval",
```

Comment:
Actually I think these are related to the old non-nested telemetry config options - you probably don't need any Circonus stuff here.